boat-quay/boat-quay-rules/src/main/kotlin/com/backbase/oss/boat/quay/ruleset/InfoBlockDescriptionChecker.kt

```diff
@@ -0,0 +1,23 @@
+package com.backbase.oss.boat.quay.ruleset
```

Comment:
Can  you also check for presence of markdown inside the description?  (make it configurable)

Reply:
I am not sure I can check if there is a markdown inside. I can check some patterns, 
https://stackoverflow.com/questions/24686588/how-to-test-if-a-string-has-markdown-in-it

Reply:
not a requirement from API Portal as far as I know @jbeerdev ?